### ISSUE TITLE
NVDA menu and settings options either missing hotkeys or hotkey not visually underlined.

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#6741]
This issue initially only covered "Report formatting changes after the cursor".  For completeness, I have recorded below all NVDA menu and settings options which either do not have a hotkey or for which the hotkey is not underlined.

When you press NVDA+n and bring up the NVDA menu, the hotkey (the letter you can press to activate that item) is underlined on most options.  For instance, the "C" in "Configuration profiles" is underlined, indicating you can press C to directly open the configuration profiles screen.

In the NVDA menu and sub-menus, there are several options where the hotkey is not underlined in the menu item and should be.

In the main NVDA menu, "Tools" should have "T" underlined, and "Donate" should have "D" underlined.

In the Tools menu, ONLY "Manage add-ons" has a letter underlined ("a"), so the rest need their hotkey letter underlined.

In the Help menu, the "About" option should have the "A" underlined.

In the "Settings" dialog (looking at the new combined multi-settings dialog), the following options do not have hotkeys underlined:
- General Page: "Use NVDA on the Windows logon screen"
- Speech Page: "Automatic language switching",
	"Automatic dialect switching"
	"Trust Voice's language..." and
	"Capital pitch change percentage"
- Braille page: "Blind cursor"
	"Cursor blink rate" and 
	"Focus context presentation"
- Keyboard page: "Use Capslock as an NVDA modifier"
	"Use numpad Insert as an NVDA modifier"
	"Use extended Insert as an NVDA modifier"
	"Speech interrupt for typed characters"
	"Speech interrupt for Enter key" and
	"Beep if typing lowercase letters when caps lock is turned on"
- Review cursor: "Simple review mode"
- Object presentation: "Report background progress bars"
- Browse mode: "Automatic focus mode for focus changes"
	"Automatic focus mode for caret movement" and
	'Audio indication of focus and browse modes"
- Document formatting: "Cell borders" and
	"Report formatting changes after the cursor"
	
And for completeness:
- In the Add-ons manager, there is no hotkey displayed for the "Installed add-ons" list itself.
- In the Create portable copy dialog, the Browse button does not have a hotkey.

**Comment by ehollig** [comment]
I also notice that the Cell borders combo box does not have a hotkey either. Suggestions on what they should be?

**Comment by feerrenrut** [comment]
Even though I believe the "report formatting changes after the cursor" option usage is discouraged, for consistency it, and the "cell borders" combo box should have an accelerator. I don't think either of these missing accelerators are particularly important, I'll label this as a P3 Bug and good for new dev. 

### Next steps:
List out the accelerators that are already used in this dialog, and suggest accelerator keys for these two options.

### Code:
Is in `source/gui/settingsDialogs.py` in `class DocumentFormattingDialog`
Add an ampersand (`&`) in front of the letter chosen to be an accelerator.
